python/pyspark/sql/pandas/map_ops.py

```diff
@@ -43,16 +43,14 @@ def mapInPandas(
     ) -> "DataFrame":
         """
         Maps an iterator of batches in the current :class:`DataFrame` using a Python native
-        function that takes and outputs a pandas DataFrame, and returns the result as a
-        :class:`DataFrame`.
+        function that is performed on pandas DataFrames both as input and output, 
```

Comment:
Let's also fix mapInArrow docs too

Reply:
Thank you! It's fixed https://github.com/apache/spark/pull/46088